### ISSUE TITLE
[htmlmaker] Hidden characters convert as spaces

### ISSUE THREAD

**Issue by nelliemckesson** [macmillanpublishers/bookmaker#128]
Some hidden characters in the Word document convert as spaces in the HTML. See the CN paragraph "Chapter 13" in this file: https://www.dropbox.com/s/hk5xc9ddrak4wgf/9781250119926_space-in-chapter-number.docx?dl=0

**Comment by nelliemckesson** [comment]
Another instance of this: https://www.dropbox.com/s/nin4ic7juieeyba/Hinshaw_Another%20Kind%20of%20Madness_clean.docx?dl=0

**Comment by nelliemckesson** [comment]
Moved both example files here: https://www.dropbox.com/sh/qi00da2khsn0a79/AAAd7xgfT1vMZmhfW1LLbHT8a?dl=0

**Comment by ericawarren** [comment]
@nelliemckesson Are you sure these are the original files and not cleaned-up versions? I can't find any additional characters in either file, even with hidden text showing. VBA doesn't count any extra characters either.

**Comment by nelliemckesson** [comment]
Yep, those are the originals. It's some kind of mysterious thing that only shows when when you convert to xml (that's why I put the issue in bookmaker rather than Word-template).

**Comment by ericawarren** [comment]
Are all the instances of this that you've seen been for CN paragraphs? Do you have the bookmaker html available somewhere?

**Comment by ericawarren** [comment]
:tada: Found it! for future reference: Word tracks previous revisions made with Track Changes _even if the changes have been accepted_, and in the XML separates them into individual text runs (i.e. `<w:r>` elements) with an `w:rsidR` attribute, which is a random number identifier of that revision. So a paragraph that has had many edits at one point may look like this in the XML:
``` xml
<w:p w14:paraId="378DAA5C" w14:textId="4A8264F1" w:rsidR="009A4DCE" w:rsidRPr="00C775C6"
    w:rsidRDefault="00CA5DA5" w:rsidP="005A6146">
    <w:pPr>
        <w:pStyle w:val="ChapTitlect"/>
    </w:pPr>
    <w:r w:rsidRPr="00C775C6">
        <w:t xml:space="preserve">The </w:t>
    </w:r>
    <w:r w:rsidR="00D708A0" w:rsidRPr="00C775C6">
        <w:t xml:space="preserve">View from </w:t>
    </w:r>
    <w:r w:rsidR="00F24D80" w:rsidRPr="00C775C6">
        <w:t>Right Field</w:t>
    </w:r>
</w:p>
```
Luckily Word 2013 has a setting buried in the Options menu to remove these: 

* **File** tab > **Options** sidebar menu item
* **Options** window > **Trust Center** sidebar > **Trust Center Settings...** button)
* **Trust Center** window > **Privacy Options** sidebar > **Document-specific settings** subheading > **Store random numbers to improve Combine accuracy** checkbox

UNCHECK this option, save the file, and 💥 , your paragraph now has this totally reasonable structure:
``` xml
<w:p>
    <w:pPr>
        <w:pStyle w:val="ChapTitlect"/>
    </w:pPr>
    <w:r>
        <w:t>The View from Right Field</w:t>
    </w:r>
</w:p>
```

I am going to look into the best way to change this programmatically, and then we can add that to all Bookmaker runs. Might be worth discussing if we should run this for all manuscripts at some point (perhaps as part of designer macro?).

**Comment by ericawarren** [comment]
Fixed by [bookmaker_addons#101](https://github.com/macmillanpublishers/bookmaker_addons/pull/101)
